wdl/src/main/scala/wdl/values/WdlCallOutputsObject.scala

```diff
@@ -0,0 +1,12 @@
+package wdl.values
+
+import wdl.WdlCall
+import wdl.types.WdlCallOutputsObjectType
+import wom.values.{WdlObjectLike, WdlValue}
+
+case class WdlCallOutputsObject(call: WdlCall, outputs: Map[String, WdlValue]) extends WdlValue with WdlObjectLike {
+  val wdlType = WdlCallOutputsObjectType(call)
+  val value = outputs
+
+  override def copyWith(values: Map[String, WdlValue]) = copy(call, values)
+}
```

Comment:
make this an implementation of `WdlObjectLike` so WOM code could treat this without knowing its true WDLy nature

Reply:
What would wom code want with a `WdlObjectLike`?

Reply:
`WdlValue.takeMaxElements` is nosy like that